src/integrationtest/java/labapi/App.java

```diff
@@ -29,4 +36,61 @@
 
     @JsonProperty("clientSecret")
     String clientSecret;
+
+    public static App convertJsonToObject(String json) throws IOException {
+
+        if (json != null) {
+            JsonFactory jsonFactory = new JsonFactory();
+            JsonParser jsonParser = jsonFactory.createParser(json);
+
+            if (jsonParser.nextToken().equals(JsonToken.START_ARRAY)) {
```

Comment:
Should this, an other usages of `JsonToken.START_ARRAY`, in integration tests be checking for `JsonToken.START_OBJECT` instead?